rolling/distribution.yaml

```diff
@@ -2651,6 +2651,23 @@ repositories:
       url: https://github.com/neobotix/neo_simulation2.git
       version: main
     status: maintained
+  nerian_stereo_ros2:
+    doc:
+      type: git
+      url: https://github.com/nerian-vision/nerian_stereo_ros2.git
+      version: master
+    release:
+      packages:
+      - nerian_stereo
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/nerian-vision/nerian_stereo_ros2-release.git
```

Comment:
The best practices for Rolling releases is to use a release repository in the [https://github.com/ros2-gbp](https://github.com/ros2-gbp) organization. That way this package can be automatically released from Rolling into the next stable ROS distribution. Please check instructions at  [https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md](https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md) describing how to create one.